ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/fonts/PaywallFontFamily.kt

```diff
@@ -0,0 +1,10 @@
+package com.revenuecat.purchases.ui.revenuecatui.fonts
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+/**
+ * Represents a font family. You can add one ore more [PaywallFont] with different weights and font styles.
+ */
+@Parcelize
+data class PaywallFontFamily(val fonts: List<PaywallFont>) : Parcelable
```

Comment:
We could probably not have this and just have a list of fonts... But seemed like a better idea to have a wrapper to mimick what it looks like for Google.